src/sass/pages/index.scss

```diff
@@ -0,0 +1,44 @@
+.home-hero {
+  @include corporate-gradient;
+  color: $color-hero-text;
+  padding: $padding-hero-vertical 0;
+
+  &__wrap {
+    @include row;
+    @include container;
+  }
+
+  &__info {
+    @include columns(5);
+
+    h1 {
+      font-size: $font-size-title-big;
```

Comment:
I'm waiting for the line-heights of all the things. If all of them are the same, yes, I will extract this to a mixin. Right now, I lack information to do that :(